### PR TITLE
Added how to sponsor page

### PR DESCRIPTION
**Pull request by rlee05**
https://www.pivotaltracker.com/story/show/133400307

### CODE REVIEW

File: how_to_sponsor.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        </header>
+    	<section id="sponsors" class="sponsors">
+            <div class="container">
+                <div class="row col-lg-12 text-center">
+                    <div class="section-heading">
+                        <h1>How to Become a Sponsor</h1>
+                        <hr class="primary">
+                    </div>
+                </div>
+                <p style="text-align: center; font-size: 1.5em;">This amazing learning opportunity would not be possible without the fantastic support of our sponsors. We strive to maintain long-term, positive relationships with companies that endorse us. If you want to be a part of this community and if the idea of helping tech enthusiasts to build incredible projects excites you, then give us a shout!</p>
```
Comment (AdrianMachado) [inline how_to_sponsor.html]: > community and if 

Insert comma after community

File: how_to_sponsor.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                        
+
+                <p style="text-align: center; font-size: 1.5em;">Contact the KW chapter at <a href="mailto:waterloo@wearhacks.com">waterloo@wearhacks.com</a> to find our how to help and how we can help you.</p>
+
+                <p style="text-align: center; font-size: 1.5em;">Find out more about our global involvement at <a href="https://wearhacks.com">wearhacks.com</a></p>
+            </div>
+        </section>
+    </body>
+</html>
```
Comment (AdrianMachado) [inline how_to_sponsor.html]: Add newline at the end of file

File: how_to_sponsor.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                <div class="row col-lg-12 text-center">
+                    <div class="section-heading">
+                        <h1>How to Become a Sponsor</h1>
+                        <hr class="primary">
+                    </div>
+                </div>
+                <p style="text-align: center; font-size: 1.5em;">This amazing learning opportunity would not be possible without the fantastic support of our sponsors. We strive to maintain long-term, positive relationships with companies that endorse us. If you want to be a part of this community and if the idea of helping tech enthusiasts to build incredible projects excites you, then give us a shout!</p>
+                        
+
+                <p style="text-align: center; font-size: 1.5em;">Contact the KW chapter at <a href="mailto:waterloo@wearhacks.com">waterloo@wearhacks.com</a> to find our how to help and how we can help you.</p>
```
Comment (AdrianMachado) [inline how_to_sponsor.html]: > our how to help and

*out
also put a comma after the help

**Review by AdrianMachado — CHANGES_REQUESTED** [review]
fix dis

**Comment by albertghionea** [comment]
Will require slight changes to the css/js file paths post-refactor. example: creative.css will no longer exist (merged into styles.css)

Nice work getting it done so quickly

**Comment by rlee05** [comment]
@albertghionea so I'll just keep creative.css in for now?

**Comment by AdrianMachado** [comment]
@rlee05 Wait till Albert's PR is approved, pull master, then fix it to work with styles
Edit: @rlee05 No thumbs pls, Hooray reacts only

**Comment by timmui** [comment]
@rlee05 can you actually move the CSS you need into a seperate file so it will be easier to move into @albertghionea's changes later?

File: how_to_sponsor.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+                </div>
+
+                <!-- Collect the nav links, forms, and other content for toggling -->
+                <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                    <ul class="nav navbar-nav navbar-right">
+              <!--  <li>
+                    <a class="page-scroll" href="#home">Home</a>
+                    </li> -->
+                    <li>
+                        <a class="page-scroll" href="#page-top">Attend</a>
```
Comment (timmui) [inline how_to_sponsor.html:29]: Shouldn't this link back to the index page instead of anchors on this page?

File: how_to_sponsor.html (hunk 2 above)
Comment (timmui) [reply how_to_sponsor.html]: There is a new line already, theres a symbol if theres no new line in github

File: how_to_sponsor.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
```
Comment (timmui) [inline how_to_sponsor.html:1]: Should probably find a place in `index.html` to link back to this page and include that as part of this PR or this page would never bee seen

**Review by timmui — CHANGES_REQUESTED** [review]

**Comment by rlee05** [comment]
@timmui like just make a css that includes the things I need?

**Comment by albertghionea** [comment]
Things that will change post-refactor:

<link rel="stylesheet" href="stylesheets/_base.css" type="text/css" /> will be removed; _base.css will no longer exist.

<link rel="stylesheet" href="stylesheets/css/creative.css" type="text/css" /> will be removed; creative.css will no longer exist; it is merged into styles.css.

<link rel="stylesheet" href="stylesheets/css/styles.css" type="text/css" /> will become <link rel="stylesheet" href="stylesheets/styles.css" type="text/css" /> ; the css folder is gone since it was redundant.

Bootstrap and fonts under bower components are being kept so those are ok.

@rlee05 if you created css classes/ids specifically for this page, a separate css file would make sense. If not, no need

**Comment by timmui** [comment]
@albertghionea When you merge in #11 , fix and merge this in too

**Pull request merged**